### ISSUE TITLE
Accessibility of OcSearchBar

### ISSUE THREAD

**Issue by kulmann** [owncloud/owncloud-design-system#1160]
The OcSearchBar components currently lacks two things regarding accessibility:
- it needs a focus border style
- the `clear` button (small `x`, only visible when the searchbar has input) is not accessible via keyboard
